lib/src/transformer/transformer.dart

```diff
@@ -14,7 +14,11 @@ import '../asset/writer.dart';
 import '../builder/builder.dart';
 import '../builder/build_step_impl.dart';
 
-abstract class BuilderTransformer implements Transformer, DeclaringTransformer {
+/// A [Transformer] which runs multiple [Builder]s. Extend this class and define
```

Comment:
uber nit: line break after the first sentence here, per [Effective Dart](https://www.dartlang.org/effective-dart/documentation/#do-make-the-first-sentence-a-standalone-paragraph)